requirements.txt

```diff
@@ -59,11 +59,11 @@ django-debug-toolbar==1.3.2
 django-waffle==0.10.1
 django-jenkins==0.17.0
 django-smoketest==0.3.0
-django-extensions==1.1.0
+django-extensions==1.5.2
 django-stagingcontext==0.1.0
 django-impersonate==0.9.1
 django-registration-redux==1.1
```

Comment:
Yes the wiki says that. However when I try to upgrade from 1.1 to 1.2 I get a "django.core.exceptions.AppRegistryNotReady: Models aren't loaded yet." error.

I started looking at the other applications that I know were upgraded to Django-1.8 to see if I could figure out why, or if there was some other change I missed but in the other apps I know of using django-1.8, they are not using registration-redux-1.2, or they are not using registration. The apps I know of using Django 1.8 are - Worth2 (uses 1.1), DMT/PMT (doesn't seem to be using registration), and Plexus (doesn't seem to be using registration).

I am looking into why I'm getting that error, but I figured it would be best to update what can be updated in the meantime. Is there another app that has been updated to 1.8 and is using registration 1.2 that I can reference?